gpu_bdb/bdb_tools/readers.py

```diff
@@ -99,7 +99,10 @@ def show_tables(self):
         return self.table_path_mapping.keys()
 
     def read(self, table, relevant_cols=None, **kwargs):
-        import dask_cudf
+        if os.getenv("CPU_ONLY") == 'True':
+            import dask.dataframe as dask_cudf
+        else:
+            import dask_cudf
```

Comment:
I would have us pass a ** kwarg and expand this class . 

https://github.com/rapidsai/gpu-bdb/blob/db207a17b2a69c6b9cb5d9183739884b2fc4a234/gpu_bdb/bdb_tools/readers.py#L90-L97 

So something like: 


```python
 def __init__( self, basepath, split_row_groups=False,  backend = dask_cudf): 
     self.table_path_mapping = { 
         table: os.path.join(basepath, table, "*.parquet") for table in TABLE_NAMES 
     } 
     self.split_row_groups = split_row_groups 
    self.back_end = back_end
```

And  Call it like : 
```
self.back_end.read_parquet
```

Reply:
Wouldn't we want the opposite or did you mean to put 'GPU' instead of 'CPU'? @VibhuJawa 

```
if back_end =='CPU'
      self.back_end = dask.dataframe
else:
      self.back_end = dask_cudf
```

Reply:
yup. Exactly that.